### PR TITLE
Enter festival name

### PR DESCRIPTION
**Pull request by eripheebs**

### CODE REVIEW

**Comment by robota-x** [comment]
frontend/www/lib is added in gitignore but all the files are in the commit, try to run the
`git rm -r --cached .`
`git add .`
`git commit -m "Removing all files in .gitignore"` commands again

**Comment by robota-x** [comment]
rest looks awesome

File: test/karma.conf.js [hunk 1]
```diff
@@ -4,9 +4,12 @@ module.exports = function(config) {
 
     frameworks: ['jasmine'],
     files: [
-      'frontend/www/lib/angular/angular.js',
-      'frontend/www/lib/angular/angular-mocks/angular-mocks.js',
-      'frontend/www/js/**/*.js',
+      '../frontend/www/lib/angular/angular.js',
+      '../frontend/www/lib/ionic/js/ionic.bundle.js',
+      '../frontend/www/lib/angular-mocks/angular-mocks.js',
+      // '../frontend/www/lib/angular-route/angular-route.js',
```
Comment (robota-x) [inline test/karma.conf.js:10]: commented code to be removed

**Comment by jazzygasper** [comment]
:shipit:

**Comment by robota-x** [comment]
:shipit:

**Comment by JojSh** [comment]
:shipit:

**Pull request merged**